### PR TITLE
Bugfix/pkgconfiglibdir

### PR DESCRIPTION
**Pull request by andrewyatz**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
- [ ] Have you successfully run `brew tests` with your changes locally?

-----
Hi. I'm proposing these changes after an error was found trying to compile graphviz on a Linux machine (see Linuxbrew/homebrew-core#3366. The inclusion of `/usr/lib/pkg-config` in `determine_pkg_config_libdir` from `ENV/super.rb` meant a system install GD library was compiled into graphviz instead of the brew installed GD library.

As asked by @sjackman I've created this PR to enact the changes he proposed to move the declaration to `os/mac/ENV/super.rb` and `homebrew_extra_pkg_config_paths`. Order of pkg-config has been retained during the move.

### CODE REVIEW

File: Library/Homebrew/extend/os/mac/extend/ENV/super.rb [hunk 1]
```diff
@@ -28,7 +28,7 @@ def homebrew_extra_paths
   # @private
   def homebrew_extra_pkg_config_paths
     paths = \
-      ["#{HOMEBREW_LIBRARY}/Homebrew/os/mac/pkgconfig/#{MacOS.version}"]
+      ["/usr/lib/pkg-config", "#{HOMEBREW_LIBRARY}/Homebrew/os/mac/pkgconfig/#{MacOS.version}"]
```
Comment (sjackman) [inline Library/Homebrew/extend/os/mac/extend/ENV/super.rb]: Should be `/usr/lib/pkgconfig` rather than `/usr/lib/pkg-config`.

**Comment by andrewyatz** [comment]
Sorry about the mistake. I've just done the change.

**Comment by MikeMcQuaid** [comment]
Thanks for your first contribution to Homebrew, @andrewyatz! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by sjackman** [comment]
Thanks for this PR, Andrew, and thanks for merging, Mike!

**Pull request merged**